### ISSUE TITLE
 Malformed UTF-8 characters, possibly incorrectly encoded with local installation

### ISSUE THREAD

**Issue by magicsepp** [contao/contao-manager#635]
[14-Feb-2021 18:25:17 UTC] [2021-02-14 18:25:17] app.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Malformed UTF-8 characters, possibly incorrectly encoded" at phar://D:/laragon4/www/cto44/web/contao-manager.phar.php/vendor/symfony/http-foundation/JsonResponse.php line 169 {"exception":"[object] (InvalidArgumentException(code: 0): Malformed UTF-8 characters, possibly incorrectly encoded at phar://D:/laragon4/www/cto44/web/contao-manager.phar.php/vendor/symfony/http-foundation/JsonResponse.php:169)"} []

**Comment by fritzmg** [comment]
You need to provide more information and not just create a new issue with only a log entry and no context what so ever.

**Comment by magicsepp** [comment]
the local update ended with error 500 and the hint for the log entry - no more information given by the manager ;(
btw - the same update online works well w/o cloud resolver (cURL error 23: Failed writing body CM 1.4.0)

**Comment by aschempp** [comment]
Sorry, but that's not enough context still. Please try to describe **all** steps you made (online and offline), and include screenshots of the problem if you can.  For example, your initial issue description didn't even say you were trying to update a Contao installation. Now I don't know what "local" or "online" is for you, and then you're saying it works well but curl Error 23?

**Comment by magicsepp** [comment]
local is an windows laragon installation - online is a webserver running the same installtion - the issue occur after updating one bundle without using the cloud resolver. With cloud resolver I got the cURL error 23) - Hope this explains the situatuion.
At the moment I got an Error 500 after selct update ro contao 4.9.12 with Config file "D:\laragon4\www\cto44\contao-manager/composer-update.get.json" is not readable or does not exist. The fie exist and ther e where diffrent contents in the file...
1st:{"cwd":"D:\\laragon4\\www\\cto44","timeout":null,"idleTimeout":null,"pid":15696,"status":"started"}
2nd:{"cwd":"D:\\laragon4\\www\\cto44","timeout":null,"idleTimeout":null,"pid":null,"status":"terminated","exitcode":0,"signaled":false,"termsig":0,"stopped":false,"stopsig":0,"timedout":0}
Finaly the file exist and the content where changed, but maybe this is an other issue.
Thanks
